src/main/java/com/undertheriver/sgsg/foler/service/FolderService.java

```diff
@@ -30,7 +33,12 @@ public Long save(Long userId, FolderDto.CreateFolderReq req) {
 			throw new IndexOutOfBoundsException(
 				String.format("폴더는 최대 %d개까지 생성할 수 있습니다!", limit));
 		}
-		return folderRepository.save(req.toEntity()).getId();
+
+		Folder folder = folderRepository.save(req.toEntity());
+		User loginUser = userService.findById(userId);
+		folder.setUser(loginUser);
```

Comment:
```suggestion
		User login = userService.findById(userId);
		Folder folder = folderRepository.save(req.toEntity(), user);
```
폴더 생성자에서 유저를 넣으면 안되나요..? 따로 세터를 쓸 필요가 없을 것 같네요.